*** funtools-1.4.4/funim.c	2007-06-14 16:08:25.000000000 -0400
--- funtools-1.4.4.new/funim.c	2012-01-13 14:24:28.000000000 -0500
***************
*** 111,119 ****
    /* optional value column for binning */
    if( fun->vbin >= 0 ){
      dovcol = 1;
!     voffset = fun->header->table->col[fun->bin[2]].offset;
!     vsize = fun->header->table->col[fun->bin[2]].size;
!     vtype = fun->header->table->col[fun->bin[2]].type;
    }
    else{
      dovcol = 0;
--- 111,119 ----
    /* optional value column for binning */
    if( fun->vbin >= 0 ){
      dovcol = 1;
!     voffset = fun->header->table->col[fun->vbin].offset;
!     vsize = fun->header->table->col[fun->vbin].size;
!     vtype = fun->header->table->col[fun->vbin].type;
    }
    else{
      dovcol = 0;
